src/redux/store.js

```diff
@@ -17,14 +17,23 @@ if (process.env.REACT_APP_ENV === 'dev') {
   compose = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || _compose; // eslint-disable-line
 }
 
-const store = createStore(
-  combineReducers({
-    ...reducers,
-    router: routerReducer,
-    form: reduxFormReducer,
-  }),
-  compose(applyMiddleware(...middlewares))
-);
-sagaMiddleware.run(rootSaga);
-export { store, history };
-window.store = store;
+const createStore = () => {
+  const store = _createStore(
+    combineReducers({
+      ...reducers,
+      router: routerReducer,
+      form: reduxFormReducer,
+    }),
+    compose(applyMiddleware(...middlewares))
+  );
+  sagaMiddleware.run(rootSaga);
+  return store;
+};
+
+const store = createStore();
+
+if (process.env.REACT_APP_ENV === 'dev') {
+  window.store = store;
```

Comment:
this does not need to be in prod env? please test with a prod build with Electron.

Reply:
this does not. I added this for testing purposes only so you could check the store's state in the console

Reply:
gotcha, so you added the window.store = store before.

Reply:
correct